compiler/rustc_session/src/options.rs

```diff
@@ -432,11 +432,11 @@ mod parse {
     /// Use this for any boolean option that has a static default.
     pub(crate) fn parse_bool(slot: &mut bool, v: Option<&str>) -> bool {
```

Comment:
should also implement this for `parse_opt_bool`

Reply:
excellent idea, ~~time to write another MCP for that as well~~